Added new kb article tooltip-show-on-disabled-element (#2607)
telerik/blazor-docs: knowledge-base/tooltip-show-on-disabled-element.md

---

> title: Displaying Tooltip on Disabled HTML Element

Reviewer: Consider an infinitive form of the verb for shortness and SEO

> description: Learn how to show a tooltip for a disabled element by wrapping it with an HTML element in Blazor applications.

Reviewer: ```suggestion
description: Learn how to show a Tooltip for a disabled element by wrapping it with another HTML element in Blazor applications.
```

type: how-to

> page_title: How to Displaying Tooltip on Disabled HTML Element

Reviewer: ```suggestion
page_title: How to Display Tooltip on Disabled HTML Element
```

slug: tooltip-kb-disabled-element
tags: tooltip, disabled element
res_type: kb
ticketid: 1665023
---

## Environment

<table>
	<tbody>
		<tr>
			<td>Product</td>
			<td>Tooltip for Blazor</td>
		</tr>
	</tbody>
</table>

## Description

When an HTML element is disabled, providing additional information via a Tooltip can be challenging since disabled elements do not respond to mouse events. Hence, the Tooltip cannot detect when the user hovers over a disabled element.

This knowledge base article also answers the following questions:
- How can I display a tooltip over a disabled HTML element in Blazor?
- How to display a tooltip over a disabled button?
- What is the workaround for showing tooltips on disabled elements in Blazor?
- Can a tooltip be associated with a disabled element in Blazor applications?

## Solution

> To display a Tooltip for a disabled button, wrap the disabled button within another HTML element, like a `<div>`, and then set the Tooltip to target this wrapper element. This approach allows the Tooltip to be activated when the user hovers over the area covering the disabled button.

Reviewer: <div>s are block elements

```suggestion
To display a Tooltip for a disabled button, wrap the disabled button in another HTML element, like a `<span>`, and then set the Tooltip to target this wrapper element. This approach allows the Tooltip to be activated when the user hovers over the area covering the disabled button.
```
